org-ql.el

```diff
@@ -921,6 +932,7 @@ Arguments STRING, POS, FILL, and LEVEL are according to
   "Return non-nil if current entry has children matching QUERY."
   (save-excursion
     (save-restriction
+      (widen) ;; Needed if we're inside an "ancestors" query
```

Comment:
Since `widen` has been added, we could just use `org-with-wide-buffer`, which does all 3 of these operations.

Reply:
Sure thing. Replaced in both functions.